src/bin/laminar-tester.rs

```diff
@@ -18,101 +19,123 @@ fn main() {
         .get_matches();
 
     if let Some(m) = matches.subcommand_matches("server") {
-        process_server_subcommand(m);
+        process_server_subcommand(m.to_owned());
     }
     if let Some(m) = matches.subcommand_matches("client") {
-        process_client_subcommand(m);
+        process_client_subcommand(m.to_owned());
     }
 }
 
-fn process_server_subcommand(m: &clap::ArgMatches<'_>) {
-    let host = m.value_of("LISTEN_HOST").unwrap();
-    let port = m.value_of("LISTEN_PORT").unwrap();
-    let st = m.value_of("SHUTDOWN_TIMER").unwrap();
+struct ClientConfiguration {
+    listen_host: SocketAddr,
+    destination: SocketAddr,
+    run_duration: Duration,
+    packet_ps: u8,
```

Comment:
Why just a byte? Seems the only place you're using it is as a `u64`. Might as well store it that way.